Core/include/Acts/Seeding/SeedFinderOrthogonal.ipp

```diff
@@ -0,0 +1,489 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2022 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "Acts/Seeding/SeedFilter.hpp"
+#include "Acts/Seeding/SeedFinderOrthogonalConfig.hpp"
+#include "Acts/Utilities/RangeXD.hpp"
+
+#include <cmath>
+#include <functional>
+#include <numeric>
+#include <type_traits>
+
+namespace Acts {
+enum Dim { DimPhi = 0, DimR = 1, DimZ = 2 };
+
+template <std::size_t N, typename external_spacepoint_t>
+RangeXD<N, float> validTupleOrthoRange(
```

Comment:
comment: returns a range reduced to the minimum defined by the config parameters

Reply:
Comment added.